### PR TITLE
Add xarray

### PR DESCRIPTION
**Pull request by stefanv**
/cc @shoyer @jarrodmillman 

I understand that this may require some further discussion with the xarray community and the SPEC committee, but wanted to put this here as a placeholder and reminder.

### CODE REVIEW

**Review by rabernat — APPROVED** [review]
At the latest xarray developers meeting, we unanimously approved the proposal to join scientific python as a core project. Thanks so much for including us!

File: core-projects/xarray.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "xarray"
+avatar: https://numfocus.org/wp-content/uploads/2018/09/xarray-logo-square.png
+homepage: https://xarray.dev/
+repository: https://github.com/pydata/xarray
+pypi: https://pypi.org/project/xarray
+libraries-io: https://libraries.io/pypi/xarray
+license: https://github.com/pydata/xarray/blob/main/LICENSE
+license-type: Apache 2.0
+contact: shoyer
```
Comment (rabernat) [inline core-projects/xarray.md:10]: We are searching for an alternate representative as @shoyer is quite overloaded with responsibilities. We will make a PR here with an update once we have our volunteer confirmed. For the interim, fine with leaving him on.

**Comment by jarrodmillman** [comment]
@rabernat Next steps:

- [ ] who should be the contact person
- [ ] would someone (could be the same person) be willing to serve on the steering committee (https://scientific-python.org/specs/steering-committee/)

**Comment by andersy005** [comment]
> who should be the contact person

@jarrodmillman, I was chatting with @stefanv @ the CZI summit, and he convinced me to nominate myself  :). So, I'm happy to serve as the contact person for Xarray.

**Comment by rabernat** [comment]
@andersy005 will you also serve on the steering committee?

**Comment by andersy005** [comment]
> @andersy005 will you also serve on the steering committee?


Yes... I forgot to mention it

File: core-projects/xarray.md [hunk 2]
```diff
@@ -7,5 +7,5 @@ pypi: https://pypi.org/project/xarray
 libraries-io: https://libraries.io/pypi/xarray
 license: https://github.com/pydata/xarray/blob/main/LICENSE
 license-type: Apache 2.0
-contact: shoyer
+contact: rabernat
```
Comment (rabernat) [inline core-projects/xarray.md:10]:
```suggestion
contact: andersy005
```

It's not me!

**Comment by jarrodmillman** [comment]
Sorry for the cut-and-paste error in e981e92 (I shouldn't work on PRs before coffee).

**Comment by jarrodmillman** [comment]
Thanks @andersy005 ! I will create a separate PR for the steering committee.

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**